src/amp-ads/40_Experimental_Ads/Scrollbound_Video_Ad.html

```diff
@@ -0,0 +1,263 @@
+      right: 0;
+      bottom: 0;
+    }
+  </style>
+</head>
+
+<body>
+  <!-- ## Scrollbound Animations -->
+  <!--
+    There are two types of scrollbound animations that occur in this sample:
+    **scaling** of the video container and **parallax** movement of the background image.
+
+    [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation)
+    is an experimental AMP component that uses [Web Animations API](https://www.w3.org/TR/web-animations/)
+    to support both __time__ and __scroll__ based animations.
+
+    **scroll vs. time ticker**: Setting `"ticker": "scroll"` tells
+    `amp-animation` to advance the animation timeline based on scrolling rather than time.
+
+    Normally animations have a duration defined, scrollbound animations also have
+    a duration but in pixels rather than milliseconds. Currently duration of
+    a scrollbound animation is approximately one viewport height, meaning all keyframes
+    of the animation will be hit between the time ad is fully in viewport until
+    user scrolls it partially out of the viewport.
+
+    **Scale down video to the corner when scrolling**:
+    As you have noticed, the video covers the whole ad initially and
+    after the page is scrolled, it slowly starts to scale down and move to the
+    corner reaching its target size when ad is in the middle of the view port.
+
+    To achieve that, we define 4 keyframes. Between 0 and 10% of viewport,
+    video is fullsize, between 10% and 50%, it starts scaling down until it
+    settles at half its size when in the middle of the viewport and stays the
+    same size all the way to the last keyframe.
+
+    **Parallax background image**:
+    You may have noticed that the background image of this ad has a subtle parallax
+    effect. To achieve that, we use scrollbound animation and specify a `60px` offset
+    at the last keyframe.
+  -->
+  <amp-animation layout="nodisplay" trigger="visibility">
+    <script type="application/json">
+      {
+      "ticker": "scroll",
+      "fill": "both",
+      "animations": [
+        {
+          "target": "videoContainer",
+          "keyframes": [
+            { "transform": "scale(1)" },
+            { "transform": "scale(1)", "offset": 0.1 },
+            { "transform": "scale(0.5)", "offset": 0.5 },
+            { "transform": "scale(0.5)", "offset": 1 }
+          ]
+        },
+        {
+          "target": "backgroundImg",
+          "keyframes": [
+            { "transform": "translateY(0)" },
+            { "transform": "translateY(60px)", "offset": 1 }
+          ]
+        }
+      ]
+    }
+    </script>
+  </amp-animation>
+
+  <!-- ## Ad Container -->
+  <!--
+  Top level container: It fills the parent viewport that hosts the ad.
+  -->
+  <div class="ad-container">
+    <!-- ## Video Layer -->
+    <!--
+    Create a click-to-play overlay and a video element.
+
+    As seen above, we use `amp-animation` to scale down the `videoContainer` when user
```

Comment:
s/As seen above, we use `amp-animation` to scale/The `amp-animation` scales down the ...